forward_engineering/ddlProvider/ddlProvider.js

```diff
@@ -198,19 +198,17 @@ module.exports = (baseProvider, options, app) => {
 				return '';
 			}
 
-			const allDeactivated = checkAllKeysDeactivated(index.indxKey || []);
+			const allDeactivated = checkAllKeysDeactivated(index.indxKey);
 			const wholeStatementCommented = index.isActivated === false || !isParentActivated || allDeactivated;
 			const indexType = index.indexType ? `${_.toUpper(index.indexType)} ` : '';
 			const ifNotExist = index.ifNotExist ? 'IF NOT EXISTS ' : '';
-			const name = wrap(index.indxName || '', '`', '`');
+			const name = wrap(index.indxName, '`', '`');
```

Comment:
because it's checked [here](https://github.com/hackolade/MariaDB/pull/51/files#diff-05d53f996b9ff3b5accb51d320c89861c42cdce7c46c15be0fe2c728d55a5196R197)